symphony-bdk-core/src/main/java/com/symphony/bdk/core/config/BdkConfigParser.java

```diff
@@ -20,14 +24,18 @@
     public static JsonNode parse(InputStream inputStream) throws BdkConfigException {
         JSON_MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
         YAML_MAPPER.configure(DeserializationFeature.FAIL_ON_UNKNOWN_PROPERTIES, false);
+        String content = new BufferedReader(
```

Comment:
since we have commons io as a dependency we could use IOUtils.toString(inputstream)